File: backend/src/main/java/com/dasima/drawrun/domain/course/dto/request/CourseSaveRequest.java
package com.dasima.drawrun.domain.course.dto.request;

import com.dasima.drawrun.domain.course.vo.GeoPoint;
import lombok.Data;

import java.util.List;

@Data
public class CourseSaveRequest {
    private List<GeoPoint> path; // 경로
    private String pathImgUrl;
    private String name;
    private double distance;
}

File: android/DrawRun/app/src/main/java/com/example/drawrun/data/dto/request/course/CourseSaveRequest.kt
package com.example.drawrun.data.dto.request.course

// 코스 이미지 저장 API와 분리해야 함
data class CourseSaveRequest(
//    val userId: String,
    val path: List<Point>,
    val pathImgUrl: String,
    val name: String,
    val distance: Double
)

// 경유지 데이터 클래스
data class Point(
    val longitude: Double,
    val latitude: Double,
)
